Cython/Utility/ModuleSetupCode.c

```diff
@@ -205,6 +205,10 @@
   #undef SHIFT
   #undef BASE
   #undef MASK
+  /* Compile-time sanity check that these are indeed equal.  Github issue #2670. */
+  #ifdef SIZEOF_VOID_P
+    enum { __pyx_check_sizeof_voidp = 1/(SIZEOF_VOID_P == sizeof(void*)) };
```

Comment:
Nice one. :)
Might be worth a CYTHON_UNUSED, though. Can't say if we'd get C compiler complaints otherwise.